### PR TITLE
Rename CRAN related functions

### LINKED ISSUE #299: Rename CRAN-related functions?
Proposed* (old -> new):

- `CRAN_checks()` -> `check_status_on_cran()`
- `on_CRAN()` -> `last_updated_on_cran()`

I think such renaming makes sense for a couple of reasons:

- Rarely a good idea to mix lower and upper case in function names.
- The second name makes it really difficult to figure out what this function actually does.

*I couldn't come up with good names that share a common prefix instead of a suffix.

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #299

### CODE REVIEW

**Comment by strengejacke** [comment]
@DominiqueMakowski

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # easystats 0.5.3
 
+* Following functions have been renamed (old -> new):
+
+  - `CRAN_checks()` -> `check_status_on_cran()`
+  - `on_CRAN()` -> `last_updated_on_cran()`
```
Comment (etiennebacher) [inline NEWS.md:6]: you said in #299 you wanted a common prefix, how about `cran_status()` and `cran_last_update()`

**Comment by etiennebacher** [comment]
Here's the output for the last CRAN updates:

image

Shouldn't the colors be reversed? I suppose it's better if the package was updated soon. Or don't put colors at all? Also I would put the date between parenthesis after the duration, like `bayestestR  18.7 weeks (2022-05-02)`

**Comment by DominiqueMakowski** [comment]
ah yes update bayestestR  almsot forgot about that thanks!

File: NEWS.md (hunk 1 above)
Comment (IndrajeetPatil) [reply NEWS.md:6]: Hmm, yeah, but I also like function names starting with verbs.

How about:

- `check_status_on_cran()`
- `check_previous_cran_update()`

**Comment by IndrajeetPatil** [comment]
> Shouldn't the colors be reversed?

@etiennebacher The colour is actually signalling if it's okay to update the package on CRAN given the last update. If it's too close to the previous update, then CRAN team _might_ reject it because this leaves a `NOTE` in R CMD check. 
So the green colour means enough time has passed to not worry about this issue. 

TBH, CRAN is not too strict about this, so this check is not that useful. I have submitted hot fix updates the next day and CRAN was okay with it (since I noted why this was necessary in CRAN comments). 

But, I guess, this is useful to synchronize the ecosystem-wide releases.

> Also I would put the date between parenthesis after the duration, like bayestestR 18.7 weeks (2022-05-02)

Okay, should be easy to implement this.

**Comment by strengejacke** [comment]
@DominiqueMakowski I pinged you for your comment on the names, not for updating _bayestestR_ ;-)

@IndrajeetPatil, a bit shorter would be:

* `check_cran_status()`
* `check_last_cran_update()`

@etiennebacher The idea was that red color indicates "warning, do not update yet", while green indicates "ok to update". It _could_ make sense to reverse colors, with green being "fresh on CRAN", and red meaning "long time on CRAN". But red usually indicates something with special attention, and the logic was that "short time" -> attention, don't submit yet. But it's not necessary to take action because a package is already 3 months on CRAN, it can live there years without updating. That's why I decided for this mapping of colors.

File: NEWS.md (hunk 1 above)
Comment (etiennebacher) [reply NEWS.md]: `check_previous_cran_update()` -> I would expect something related to tests and failures, not date of last update. But the original names are fine

**Comment by etiennebacher** [comment]
> The idea was that red color indicates "warning, do not update yet", while green indicates "ok to update". It _could_ make sense to reverse colors, with green being "fresh on CRAN", and red meaning "long time on CRAN". But red usually indicates something with special attention, and the logic was that "short time" -> attention, don't submit yet. But it's not necessary to take action because a package is already 3 months on CRAN, it can live there years without updating. That's why I decided for this mapping of colors.

Ah ok, makes sense ;)

**Review by strengejacke — APPROVED** [review]

**Comment by DominiqueMakowski** [comment]
how about:
```
check_cran_status()
check_cran_updated()  # it's kind of implied that it's the last time it got updated no?
```

Yes for reversing the colours, red should be warning need to update

File: NEWS.md (hunk 1 above)
Comment (IndrajeetPatil) [reply NEWS.md]:
``` r
library(easystats)
#> # Attaching packages: easystats 0.5.2.6
#> ✔ bayestestR  0.12.1.4     ✔ correlation 0.8.2.3   
#> ✔ datawizard  0.5.1.6      ✔ effectsize  0.7.0.9999
#> ✔ insight     0.18.2.8     ✔ modelbased  0.8.5     
#> ✔ performance 0.9.2.2      ✔ parameters  0.18.2.5  
#> ✔ report      0.5.5.1      ✔ see         0.7.2.2
check_last_cran_update()
#> bayestestR  18.7 weeks (2022-05-02)
#> correlation 4.6 weeks (2022-08-09)
#> datawizard  3.4 weeks (2022-08-17)
#> easystats   1.6 weeks (2022-08-30)
#> effectsize  4.4 weeks (2022-08-10)
#> insight     4.4 weeks (2022-08-10)
#> modelbased  3.3 weeks (2022-08-18)
#> performance 4.4 weeks (2022-08-10)
#> parameters  4.4 weeks (2022-08-10)
#> report      2.7 weeks (2022-08-22)
#> see         4.4 weeks (2022-08-10)
```

<sup>Created on 2022-09-10 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Pull request merged**